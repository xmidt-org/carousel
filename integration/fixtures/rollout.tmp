terraform init
terraform workspace new simple
terraform apply --auto-approve -var versionGreen=0.1.0 -var versionGreenCount=2