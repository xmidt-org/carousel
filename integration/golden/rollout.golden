not checking hosts
completed step: blue with 0 nodes and green with 2 nodes
completed step: blue with 1 nodes and green with 2 nodes
completed step: blue with 1 nodes and green with 1 nodes
completed step: blue with 2 nodes and green with 1 nodes
completed step: blue with 2 nodes and green with 0 nodes
completed step: blue with 3 nodes and green with 0 nodes
